resources/js/views/general/Tag.vue

```diff
@@ -223,23 +220,27 @@ export default {
          */
         async confirmDelete (photoid)
         {
-            if (confirm("Do you want to delete this image? This cannot be undone."))
+            if (confirm('Do you want to delete this image? This cannot be undone.'))
```

Comment:
Needs translation

Reply:
Updated. Thanks

Reply:
It actually needs a translation key, like `{{ $t('tag.confirm-delete') }}` and we need to add the translation key to each language folder. We also have a new excel sheet to track translation keys

Reply:
Yeah, I've added the translation key, did you check the new commit? I will add translation key to each language folder.